src/ext/ddtrace.c

```diff
@@ -403,6 +403,9 @@ static PHP_RINIT_FUNCTION(ddtrace) {
         return SUCCESS;
     }
 
+    // currently only use this for error.msg, error.stack, and error.type, so use 3
+    array_init_size(&DDTRACE_G(additional_trace_meta), 3);
```

Comment:
Nit: Since this magic number is used [here](https://github.com/DataDog/dd-trace-php/blob/5b01859e2bcebc13f8b66f5a444e10660d8c9d86/src/ext/php7/serializer.c#L630), [here](https://github.com/DataDog/dd-trace-php/blob/5b01859e2bcebc13f8b66f5a444e10660d8c9d86/src/ext/php7/serializer.c#L701), and I'm guessing should be used [here too](https://github.com/DataDog/dd-trace-php/blob/5b01859e2bcebc13f8b66f5a444e10660d8c9d86/src/ext/span.c#L169), perhaps we can make it a macro?